tests/test_datasets/test_dataset_functions.py

```diff
@@ -909,7 +996,7 @@ def test_create_dataset_pandas(self):
             collection_date=collection_date,
             language=language,
             licence=licence,
-            default_target_attribute=default_target_attribute,
+            default_target_attribute="play",
```

Comment:
Why did you change these tests?

Reply:
This one will fail because of the new logic , as "play" is not in the dataset columns.

Reply:
But previously it was also set to `"play"`? And it says `column_names = ["rnd_str", "outlook", "temperature", "humidity", "windy", "play"]` just a few lines up. I am a bit confused about this. Maybe sitting together for this works better.